docs/api/signal.md

```diff
@@ -193,45 +173,24 @@ function myAction ({path}) {
 To actually diverge down the path you have to call it and return it from the action:
 
 ```js
-function myAction ({path}) {
+function myAction ({ path }) {
   return path.foo()
 }
 ```
 
 Optionally pass a payload:
 
 ```js
-function myAction ({path}) {
-  return path.bananas({foo: 'bar'})
+function myAction ({ path }) {
+  return path.bananas({ foo: 'bar' })
 }
 ```
 
 With promises you just return it the same way:
 
 ```js
-function myAction ({someProvider, path}) {
+function myAction ({ someProvider, path }) {
   return someProvider.doAsync()
-    .then((result) => path.bananas({data: result.data}))
+    .then((result) => path.bananas({ data: result.data }))
 }
 ```
-
-## Errors
```

Comment:
leave and add removal note?

Reply:
We have it in migration docs, think that is enough :)